tests/tcl/tests/unit/command.tcl

```diff
@@ -0,0 +1,33 @@
+start_server {tags {"command"}} {
+    test {COMMAND count} {
+        r command count
+    } {155}
+
+    test {COMMAND info} {
+        set e [lindex [r command info get mget] 0]
+        assert_equal [llength $e] 6
+        assert_equal [lindex $e 0] get
+        assert_equal [lindex $e 1] 2
+        assert_equal [lindex $e 2] {readonly}
+        assert_equal [lindex $e 3] 1
+        assert_equal [lindex $e 4] 1
+        assert_equal [lindex $e 5] 1
+    }
+
+    test {COMMAND} {
+        set e [lindex [r command] 16]
```

Comment:
I think this case may be easy to fail because we may add new commands, right?

Reply:
may be only test length?

Reply:
i notice you deleted this case, i feel ok to keep current change